web/src/main/java/com/bazaarvoice/emodb/web/migrator/MetricsMigrationCountListener.java

```diff
@@ -0,0 +1,33 @@
+package com.bazaarvoice.emodb.web.migrator;
+
+import com.bazaarvoice.emodb.web.scanner.notifications.ScanCountListener;
+import com.codahale.metrics.Gauge;
+import com.codahale.metrics.MetricRegistry;
+import com.google.inject.Inject;
+
+public class MetricsMigrationCountListener implements ScanCountListener {
```

Comment:
Again, not sure all these listeners are necessary since there is no external consumer of the migration.  Just the metrics should be enough.